### ISSUE TITLE
Server Process Onload

### ISSUE THREAD

**Issue by gkf442573575** [pqina/filepond#72]
Server response in process onload, I return a `res.fiid`, but the value in file input is 'undefined'

This is my Server Options

```js
server: {
    url: basepath,
    timeout: 7000,
    process: {
        url: 'app/file/save.do?tpid=2',
        method: 'POST',
        crossDomain: true,
        withCredentials: true,
        onload: function onload(res) {
            console.log('res', res.fiid);

            return res.fiid;
        },
        onerror: function onerror(err) {}
    },
    revert: {
        url: 'app/file/delFile.do',
        method: 'DELETE',
        crossDomain: true,
        withCredentials: true,
        onload: res => {},
        onerror: err => {}
    }
}
```

**Comment by rikschennink** [comment]
Try this:

```js
onload: function onload(res) {
    return res.fiid + '';
},
```

**Comment by rikschennink** [comment]
Just published version `1.8.4` which should fix this without using the solution posted above.

**Comment by rikschennink** [comment]
Will close this for now.
